### PR TITLE
multiple inline formsets

### PR DESCRIPTION
**Pull request by meska**
with this commit I added multiple inline formsets and an italian translation
the change is compatible with applications with single inline formset

### CODE REVIEW

File: crudbuilder/abstract.py [hunk 1]
```diff
@@ -78,7 +79,10 @@ def check_permission_required(self):
     @property
     def get_inlineformset(self):
         if self.crud.inlineformset:
-            return self.crud.inlineformset().construct_formset()
+            if isinstance(self.crud.inlineformset, list):
```
Comment (mauridb) [inline crudbuilder/abstract.py]: @meska about your **inlineformset changes**
i wanna create an inlineformset with 2+ child model(inline_model) with the same Parent...
could your feature permit me this task?

Example:
+  Parent form
     + child form(same parent)
     + child form(same parent)

**Comment by Jcbobo88** [comment]
I notice a strange behaviour,run the example (without modification) as soon i enter in the list view of any crud i get the  get_context_data() keywords must be strings errors, 
the context_object_name is assigned to a django.utils.functional.__proxy__ instance instead of a string/unicode 
adding the .title()  call to line 87 of crudbuilder/views.py solve the issues.
running the tests all tests fail for url path issues. 

@meska I need this issue solved in brief time without translation (good work for italian :D), i will create a parallel fork just with the multiple inline feature, as soon the translation will be working correclty i will delete my fork 

@asifpy  do you think this feature can be usefull? creating a more atomic PR will be fine?

**Comment by russell310** [comment]
@meska Could you please check recent changes and update your PR accordingly?

**Comment by meska** [comment]
> @meska Could you please check recent changes and update your PR accordingly?

**Pull request closed without merging**